src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int START_POINT = 0;
+    private static final int END_POINT = 10;
+    private static final int START_POINT_CHECKING = 11;
+    private static final int PLUS_TO_INDEX = 1;
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate fromDate = LocalDate.parse(dateFrom, formatter);
+        LocalDate toDate = LocalDate.parse(dateTo, formatter);
+        fromDate = fromDate.minusDays(1);
+        toDate = toDate.plusDays(1);
+        StringBuilder sb = new StringBuilder();
+        sb.append("Report for period ")
+                .append(dateFrom)
+                .append(" - ")
+                .append(dateTo)
+                .append(System.lineSeparator());
+        int salary = 0;
+        for (int i = 0; i < names.length; i++) {
+            for (int j = 0; j < data.length; j++) {
```

Comment:
```suggestion
        for (int i = 0; i < names.length; i++) {
            int salary = 0;
            for (int j = 0; j < data.length; j++) {
```
salary will be 0 on each iteration